Modules/Devices/test/Devices.Domain.Tests/Identities/CancelStaleDeletionProcessTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using Backbone.UnitTestTools.Extensions;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelStaleDeletionProcessTests
```

Comment:
There are tests missing for the following cases:
* The deletion process id does not exist
* The deletion process is not in status "WaitingForApproval"

Reply:
Reminder: we agreed that there is no need for id test as id is not used in the method

On another note, if you remember, the tested method used to return Result and also throw the exception. I think I found a really nice way to solve that. Test is also updated accordingly.

Reply:
In your solution you just return an error message. But I think I remember saying that you should return a DomainError so that we have an error code.

Reply:
Yeah, I misunderstood you when we talked about returning Result and throwing an exception in a same method. It's fixed.